src/javascript/app/pages/trade/markets.jsx

```diff
@@ -90,6 +91,12 @@ class Markets extends React.Component {
             market_symbol = Object.keys(this.markets).find(m => this.markets[m].submarkets[market_symbol]);
             Defaults.set('market', market_symbol);
         }
+        this.keys_arr = [];
+        this.markets_all.forEach((market) => {
+            if (market[1].subgroup_name !== 'None') {
```

Comment:
Can we use `subgroup` instead of `subgroup_name` since the second one will be translated into other languages and the condition will not work?

Reply:
Resolved

Reply:
I still see `subgroup_name`

Reply:
resolved 🙏🏼